db/migrate/20190128142743_create_project_update_priorities.rb

```diff
@@ -0,0 +1,9 @@
+class CreateProjectUpdatePriorities < ActiveRecord::Migration[5.2]
+  def change
+    create_table :project_update_priorities do |t|
+      t.references :project, unique: true
+      t.integer :priority, default: 0
+      t.timestamps
+    end
```

Comment:
Can we add an index to make it unique on project id so we don't have a project in multiple priorities? If not, we should make the project + priority a unique combo.